src/lib/bootloader/grub2_widgets.rb

```diff
@@ -336,6 +338,19 @@ def init
     def store
       grub2.secure_boot = value
     end
+
+    def validate
+      return true if Yast::Mode.config ||
+        !Yast::Arch.s390 ||
+        value == Systeminfo.secure_boot_active?
+
+      Yast::Popup.ContinueCancel(
+        _(
+          "Make sure the Secure Boot setting matches the configuration of the HMC.\n\n" \
+          "Otherwise this system will not boot."
```

Comment:
Well, buttons are continue/cancel. So I expect here a bit hint to user what when they click it. Like `Cancel to edit it` or `Continue to use setting`.

Reply:
You mean like here?:

https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/grub2_widgets.rb#L388-L390

I've never seen a dialog like 'continue to continue' or 'cancel to cancel'. That's why there's a button text.